sunpy/map/sources/sdo.py

```diff
@@ -165,6 +165,15 @@ def reference_date(self):
     def _set_reference_date(self, date):
         self.meta['T_OBS'] = parse_time(date).utc.isot
 
+    @property
+    def unit(self):
+        unit_str = self.meta.get('bunit', None)
+        if unit_str is None:
+            return
+        # Maxwells aren't in the IAU unit style manual and therefore not a valid FITS unit
+        # The mapbase unit property forces this validation, so we must override it to prevent it.
+        return u.Unit(unit_str)
```

Comment:
If we do this, I feel it should be done only for the specific unit expected ("Mx / cm2") rather than allowing any unit

Reply:
Agreed. I just copied this straight from `HMISynoptic` source where we were already doing this.